pymatgen/electronic_structure/dos.py

```diff
@@ -1329,21 +1407,21 @@ def get_site_orbital_dos(self, site: PeriodicSite, orbital: str) -> Dos:  # type
             "f_xz^2",
             "f_z(x^2-y^2)",
             "f_x(x^2-3y^2)",
-        ]:
+        }:
             raise ValueError("orbital is not correct")
         return Dos(self.efermi, self.energies, self.pdos[site][orbital])  # type: ignore
 
-    def get_site_t2g_eg_resolved_dos(self, site: PeriodicSite) -> dict[str, Dos]:
-        """Get the t2g, eg projected DOS for a particular site.
+    def get_site_t2g_eg_resolved_dos(self, site: PeriodicSite) -> dict[Literal["e_g", "t2g"], Dos]:
+        """Get the t2g/e_g projected DOS for a particular site.
 
         Args:
-            site: Site in Structure associated with CompleteDos.
+            site (PeriodicSite): Site in Structure associated with LobsterCompleteDos.
 
         Returns:
-            A dict {"e_g": Dos, "t2g": Dos} containing summed e_g and t2g DOS
-            for the site.
+            {"e_g": Dos, "t2g": Dos}: Summed e_g and t2g DOS for the site.
         """
-        warnings.warn("Are the orbitals correctly oriented? Are you sure?")
+        warnings.warn("Are the orbitals correctly oriented? Are you sure?")  # TODO: DanielYang: unconditional warning
```

Comment:
There is a very strong but unconditional warning here.

Reply:
@JaGeo Do we want to make this warning less strong? Like `Please make sure the orbitals are correctly oriented.`. Or add a condition for such warning (if possible)?

Reply:
Thanks for the input! I get it now (check the bond spatial alignment relative to the global axes). Can I change the warning message a bit to make it more descriptive and less strong? Is there any downstream operations that depend on the message?

Perhaps `Please make sure the orbitals/bonds are correctly oriented with respect to global axes`? Or do you want to keep it as is?